wiki/People/The_Team/Beatmap_Nominators/en.md

```diff
@@ -210,8 +213,6 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 
 | Name | Additional languages |
```

Comment:
note: between removing the table and the heading altogether and keeping it empty I too choose the latter